src/Sets/Zonotope.jl

```diff
@@ -492,3 +492,44 @@ function quadratic_map(Q::Vector{MT}, Z::Zonotope{N}) where {N, MT<:AbstractMatr
     end
     return Zonotope(d, remove_zero_columns(h))
 end
+
+# ====================================
+# Zonotope vertex enumeration methods
+# ====================================
+
+function _vertices_list_2D(c::AbstractVector{N}, G::AbstractMatrix{N}) where {N}
+    if same_sign(G)
+        return _vertices_list_2D_positive(c, G)
+    else
+        # FIXME generalized 2D vertices list function is not implemented yet
+        # See LazySets#2209
+        return _vertices_list_iterative(c, G)
```

Comment:
Pass the `apply_convex_hull` argument to this function and then pass it on below. Otherwise the behavior is unexpected from the user perspective.
```suggestion
        return _vertices_list_iterative(c, G, apply_convex_hull=apply_convex_hull)
```

Reply:
ok :+1: 

note this is transient though, will be replaced by https://github.com/JuliaReach/LazySets.jl/pull/2288